pkgs/development/python-modules/mkdocs-autolinks-plugin/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, mkdocs
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-autolinks-plugin";
+  version = "0.7.1";
+  format = "setuptools";
+
```

`mkdocs` itself only supports Python > 3.8. I suggest to limit the package this release as the chance is very low that it will be tested with older releases.

```suggestion
  disabled = pythonOlder "3.8";


```